_includes/partners/contact.html

```diff
@@ -1,40 +1,387 @@
-<main id="main-content">
-  <section>
-    <div class="container partners-container">
-      <div class="grid-row">
-        <div class="tablet:grid-col-9 accordion-container">
-          <script charset="utf-8" type="text/javascript" src='{{ "/assets/js/jquery/3.5.1/jquery.min.js" | relative_url }}'></script>
```

Comment:
Can we remove the jQuery script. As I recall, it was only necessary for this embed.

Reply:
good catch!!